crates/re_viewer_context/src/lib.rs

```diff
@@ -52,11 +52,33 @@ impl SpaceViewId {
         Self(uuid::Uuid::new_v4())
     }
 
+    pub fn random_from_str(s: &str) -> Self {
+        // TODO(jleibs): This is overkill; we can probably get away with 64-bit ids.
+        use rand::{Rng as _, SeedableRng as _};
+        use std::hash::{Hash as _, Hasher as _};
+
+        let salt: u64 = 0x307b_e149_0a3a_5552;
+
+        let mut hasher = std::collections::hash_map::DefaultHasher::default();
+        salt.hash(&mut hasher);
+        s.hash(&mut hasher);
+        let mut rng = rand::rngs::StdRng::seed_from_u64(hasher.finish());
+        let uuid = uuid::Builder::from_random_bytes(rng.gen()).into_uuid();
```

Comment:
Good catch; I'll be sure to let this guy know since I copied the code from him:
![image](https://github.com/rerun-io/rerun/assets/3312232/f613ad50-0bdc-40d5-8aff-05121d27859c)